### PR TITLE
make start() chainable

### PR DESCRIPTION
**Pull request by sam-github**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -130,6 +130,8 @@ aspect.after(module.__proto__, 'require', data, function(obj, methodName, args,
         for (var i = 0; i < probes.length; i++) {
             if (probes[i].name === args[0]) {
                 ret = probes[i].attach(args[0], ret, module.exports);
+                // XXX(sam) can't have multiple probes with same name, why not
+                // return from the loop immediately?
             }
```
Comment (sjanuary) [inline index.js]: @sam-github we can and do have multiple probes with the same name (http and http-outbound are both named "http" as both attach to the http module.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "appmetrics",
-  "version": "2.0.0",
+  "version": "2.1.0",
```
Comment (sjanuary) [inline package.json]: @sam-github we haven't released 2.0.0 yet...

**Comment by sam-github** [comment]
@sjanuary PTAL

**Comment by sjanuary** [comment]
Now merged

**Pull request merged**